tornjak-frontend/src/tables/table-body.tsx

```diff
@@ -57,7 +57,7 @@ class Body extends React.Component<BodyProp, BodyState> {
                                     cell.value)}
                             </TableCell>
                         ))}
-                        {((this.props.entityType === "Agent" && this.TornjakHelper.checkRolesAdminUser(this.props.globalUserRoles)) || (this.props.entityType && !Auth_Server_Uri)) &&
+                        {((this.props.entityType === "Agent" && this.TornjakHelper.checkRolesAdminUser(this.props.globalUserRoles)) || (this.props.entityType === "Agent" && !Auth_Server_Uri)) &&
```

Comment:
This looks like the fix for the issue #149 if so, we should update the PR with "It resolves issue #149" and if testing successful, please close the issue.